src/l500/l500-depth.cpp

```diff
@@ -174,7 +174,8 @@ namespace librealsense
         tags.push_back({ RS2_STREAM_DEPTH, -1, width, height, RS2_FORMAT_Z16, 30, profile_tag::PROFILE_TAG_SUPERSET | profile_tag::PROFILE_TAG_DEFAULT });
         tags.push_back({ RS2_STREAM_INFRARED, -1, width, height, RS2_FORMAT_Y8, 30, profile_tag::PROFILE_TAG_SUPERSET | profile_tag::PROFILE_TAG_DEFAULT });
         tags.push_back({ RS2_STREAM_CONFIDENCE, -1, width, height, RS2_FORMAT_RAW8, 30, profile_tag::PROFILE_TAG_SUPERSET });
-        
+        tags.push_back({ RS2_STREAM_DEPTH, -1, 800, 600, RS2_FORMAT_FG, 30, profile_tag::PROFILE_TAG_DEBUG } );
```

Comment:
Can you please contact Miri/Aviad and ask how USB2 affects the FG stream, if at all?

Reply:
FG not supported on usb2

Reply:
Then shouldn't this not add them under USB2?

Reply:
Its not exposed by FW we should not worry for this

Reply:
But we shouldn't do this per resolution, right?